### ISSUE TITLE
Bug: correct eslint tooling

### ISSUE THREAD

**Issue by 0-vortex** [open-sauced/hot#282]
### Describe the bug

Eslint fixing rules works very well after #269 however there are some issues that were hanging for a while:
- vite eslint cache issue
- IDE auto-config for vsc, gitpod, etc

### Steps to reproduce

1. Clone the project
2. Run `npm ci && npm start` in a terminal window/vsc/intellij/etc
3. Introduce a linting error and check the terminal window
4. Correct code linting error and check the terminal, the error should still be there

### Affected services

hot.opensauced.pizza

### Additional context

auto-run the formatter on commit (could be opened as a separate issue if it proves difficult)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by 0-vortex** [comment]
I'll figure this out over the weekend, want to use gitpod and more vsc for tutorial purposes 🍕
